Sources/Foundation/CMakeLists.txt

```diff
@@ -0,0 +1,168 @@
+    ISO8601DateFormatter.swift
+    JSONDecoder.swift
+    JSONEncoder.swift
+    JSONSerialization.swift
+    JSONSerialization+Parser.swift
+    LengthFormatter.swift
+    MassFormatter.swift
+    Measurement.swift
+    MeasurementFormatter.swift
+    Morphology.swift
+    Notification.swift
+    NotificationQueue.swift
+    NSArray.swift
+    NSAttributedString.swift
+    NSCache.swift
+    NSCalendar.swift
+    NSCFArray.swift
+    NSCFBoolean.swift
+    NSCFCharacterSet.swift
+    NSCFDictionary.swift
+    NSCFSet.swift
+    NSCFString.swift
+    NSCFTypeShims.swift
+    NSCharacterSet.swift
+    NSCoder.swift
+    NSComparisonPredicate.swift
+    NSCompoundPredicate.swift
+    NSConcreteValue.swift
+    NSData.swift
+    NSData+DataProtocol.swift
+    NSDate.swift
+    NSDateComponents.swift
+    NSDecimalNumber.swift
+    NSDictionary.swift
+    NSEnumerator.swift
+    NSError.swift
+    NSExpression.swift
+    NSGeometry.swift
+    NSIndexPath.swift
+    NSIndexSet.swift
+    NSKeyedArchiver.swift
+    NSKeyedArchiverHelpers.swift
+    NSKeyedCoderOldStyleArray.swift
+    NSKeyedUnarchiver.swift
+    NSLocale.swift
+    NSLock.swift
+    NSLog.swift
+    NSMeasurement.swift
+    NSNotification.swift
+    NSNull.swift
+    NSNumber.swift
+    NSObjCRuntime.swift
+    NSObject.swift
+    NSOrderedSet.swift
+    NSPathUtilities.swift
+    NSPersonNameComponents.swift
+    NSPlatform.swift
+    NSPredicate.swift
+    NSRange.swift
+    NSRegularExpression.swift
+    NSSet.swift
+    NSSortDescriptor.swift
+    NSSpecialValue.swift
+    NSString.swift
+    NSStringAPI.swift
+    NSSwiftRuntime.swift
+    NSTextCheckingResult.swift
+    NSTimeZone.swift
+    NSURL.swift
+    NSURLComponents.swift
+    NSURLError.swift
+    NSURLQueryItem.swift
+    NSUUID.swift
+    NSValue.swift
+    NumberFormatter.swift
+    Operation.swift
+    PersonNameComponents.swift
+    PersonNameComponentsFormatter.swift
+    Port.swift
+    PortMessage.swift
+    Process.swift
+    ProcessInfo.swift
+    Progress.swift
+    ProgressFraction.swift
+    PropertyListSerialization.swift
+    ReferenceConvertible.swift
+    RunLoop.swift
+    Scanner.swift
+    ScannerAPI.swift
+    Set.swift
+    Stream.swift
+    String.swift
+    StringEncodings.swift
+    Thread.swift
+    Timer.swift
+    Unit.swift
+    URL.swift
+    URLComponents.swift
+    URLQueryItem.swift
+    URLResourceKey.swift
+    UserDefaults.swift
+    UUID.swift
+    WinSDK+Extensions.swift)
+
+target_compile_options(Foundation PRIVATE
+    "SHELL:$<$<COMPILE_LANGUAGE:Swift>:${_Foundation_swift_build_flags}>")
+
+target_link_libraries(Foundation
+    PUBLIC
+        CoreFoundation
+        FoundationEssentials
+        FoundationInternationalization)
+
+set_target_properties(Foundation PROPERTIES
+    INSTALL_RPATH "$ORIGIN"
+    BUILD_RPATH "$<TARGET_FILE_DIR:swiftDispatch>"
+    Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift
+    INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_BINARY_DIR}/swift)
```

Comment:
We may need to massage this a bit. `${CMAKE_BINARY_DIR}` is an absolute path, and that will give written into the install scripts.

Worth reading https://cmake.org/cmake/help/latest/command/target_include_directories.html#creating-relocatable-packages

Reply:
I ended up just removing these.